convert/convert.go

```diff
@@ -169,7 +172,8 @@ func getWigChromIndex(s string, wigSlice []wig.Wig) int {
 	return -1
 }
 
-//PairwiseFaToVcf takes in a pairwise multiFa alignment and writes Vcf entries for segregating sites with the first entry as the reference and the second fasta entry as the alt allele.
+//PairwiseFaToVcf takes in a pairwise multiFa alignment and writes Vcf entries for segregating sites with the first
+// entry as the reference and the second fasta entry as the alt allele.
```

Comment:
Very minor, but this line has a space before the text whereas the line above does not. May be something to update in future godoc-related PRs.

Reply:
I'll just update it now. If you can re-approve that'd be great!